config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/AnnotatedEntity.java

```diff
@@ -16,13 +16,31 @@
     private String policyName;
     private String description;
     private boolean bundle;
-    private boolean reusable;
+    private boolean reusable=true;
```

Comment:
is it different from reusableEntity, if not please remove duplicates

Reply:
yes, its for bundle